libs/dao/src/main/java/com/akto/dao/test_editor/strategy/StrategyParser.java

```diff
@@ -0,0 +1,25 @@
+package com.akto.dao.test_editor.strategy;
+
+import java.util.Map;
+
+import com.akto.dto.test_editor.Strategy;
+
+public class StrategyParser {
+    
+    public Strategy parse(Object metadataObj) {
+        Map<String, Object> metadataMap = (Map) metadataObj;
+        Strategy strategy = new Strategy();
+
+        if (metadataMap == null) {
+            return strategy;
+        }
+
+        Object val = metadataMap.get("run_once");
```

Comment:
Will u do this for all the fields. Use gson or something so that if u define setter it will automatically parse